RallyHereIntegration/Source/RallyHereGameHostProvider/Private/RH_GameHostProviderGHA.cpp

```diff
@@ -373,7 +380,16 @@ void FRH_GameHostProviderGHA::OnReservationComplete(const RallyHereStatusCode& c
 		UE_LOG(LogRHGameHostProvider, Log, TEXT("[%s] GameHostAdapter successfully created reservation: %s"), ANSI_TO_TCHAR(__FUNCTION__), UTF8_TO_TCHAR(GameHostAdapterImporter::rallyhere_status_text(code)));
 	}
 
-	OnProviderReservationComplete.ExecuteIfBound(!bIsError);
+	
+	RallyHereStringMapPtr reservation_info;
```

Comment:
You want to initialize this as a nullptr so that if its not reserved your delete function doesn't do anything.